### ISSUE TITLE
Button groups do not respect the `sizing_mode` or `width` keyword arguments.

### ISSUE THREAD

**Issue by GenevieveBuckley** [bokeh/bokeh#8443]
Button groups (`RadioButtonGroup`, `CheckboxButtonGroup`) do not respect the `sizing_mode` or `width` keyword arguments.

#### ALL software version info (bokeh, python, notebook, OS, browser, any other relevant packages)
bokeh:  version 1.0.1
python: 3.7.1
jupyter notebook: 5.7.0
Chrome: version 70.0.3538.77 (Official Build) (64-bit)

#### Description of expected behavior and the observed behavior
Expected behavior: when I wrap a button group in a widgetbox/row/column/layout/etc and specify either the `sizing_mode` or `width` keyword arguments, I expect to see the button group stretch to fit the new size.
Observed behavior: I see no change in the size of widgets created with `RadioButtonGroup` or `CheckboxButtonGroup` when using these keyword arguments.

#### Complete, minimal, self-contained example code that reproduces the issue

```python
from bokeh.io import push_notebook, output_notebook, show
from bokeh.layouts import widgetbox
from bokeh.models.widgets import Button, RadioButtonGroup
output_notebook()

button = Button(label="Button")
buttongroup = RadioButtonGroup(labels=["Option 1", "Option 2", "Option 3"])

show(widgetbox(button))
show(widgetbox(button, width=500))
show(widgetbox(button, sizing_mode='scale_width'))

show(widgetbox(buttongroup))
show(widgetbox(buttongroup, width=500))
show(widgetbox(buttongroup, sizing_mode='scale_width'))
```

#### Stack traceback and/or browser JavaScript console output
There is no python or javascript error produced.

#### Screenshots or screencasts of the bug in action
This is the result from the code snippet above:
![screen shot 2018-11-16 at 2 43 33 pm](https://user-images.githubusercontent.com/30920819/48596718-63a2e000-e9ae-11e8-89b5-8dfb63a03276.png)

**Comment by GenevieveBuckley** [comment]
It also appears that there is no way to scale widgets to half of the dynamic screen size (or some other fraction), is this correct?

**Comment by bryevdv** [comment]
This is fixed in master (probably #8085):

![screen shot 2019-02-18 at 19 24 48](https://user-images.githubusercontent.com/1078448/52988129-e6bf4200-33b2-11e9-832e-050b06755b30.png)

Changing the browser window width causes the buttons to update appropriately as well.

**Comment by bryevdv** [comment]
I should also note the original code was not correct and was updated:

* re-using the same widget does not seem advised
* width should be set on the widget itself, not the widgetbox
